test/performance/profiling.md

```diff
@@ -0,0 +1,82 @@
+# Profiling Knative Serving
+
+Knative Serving allows for collecting runtime profiling data expected
+by the pprof visualization tool. Profiling data is available for the autoscaler,
+activator, controller, webhook and for the queue-proxy
+container which is injected into the user application pod. When enabled
+Knative serves profiling data on the default port 8008 through a web server.
+
+## Steps to get profiling data:
+
+* Edit the `config-observability` ConfigMap and add `profiling.enable = "true"`:
+
+    ```shell
+    kubectl edit configmap config-observability -n knative-serving
+    ```
+
+* Use port-forwarding to get access to Knative Serving pods. For example, activator:
+
+    ```shell
+    ACTIVATOR_POD=$(kubectl get pods -n knative-serving | grep activator | awk '{print $1}')
```

Comment:
```suggestion
    ACTIVATOR_POD=$(kubectl -n knative-serving get pods -l app=activator -o custom-columns=:metadata.name --no-headers)
```

You might want to add a note that this can be more than one pod.